### ISSUE TITLE
I want to provide resources translated into Japanese.

### ISSUE THREAD

**Issue by onein528** [MarcAnt01/Fluent-Screen-Recorder#51]
Hi

I would like to provide resources translated into Japanese, but the translation sources are all hard-coded. This does not comply with international standards.

Should I write a Japanese translation in this issue? I thought that this should be written in "Pull Request", but I thought that I thought that This app seem to comply with international standards.
, so I chose "Issue".

Sorry

**Comment by MarcAnt01** [comment]
Hi @onein528 I am preparing the app to be ready to be translated into other languages. Really happy to see your interest, I prepared all the string to be translated, but I have to setup the VS extension to automatically add the resources to other languages so that you can translate them. I'll let you know in some days when it's ready :)

**Comment by onein528** [comment]
I see!

**Comment by MarcAnt01** [comment]
Hi @onein528 I haven't forgot about your request, localization is done, but I'd rather to start adding translations from users only once all the app strings are there. In any case, I am working on the first major update, so even if I decide to merge those kind of pull request, you couldn't see the translated version if not in a few months.

**Comment by onein528** [comment]
Hi @MarcAnt01 
Okey, I dont care. I just thought i could do it..

Thank you.

**Comment by MarcAnt01** [comment]
> Hi @MarcAnt01
> Okey, I dont care. I just thought i could do it..
> 
> Thank you.

You can still do it, I will mention you once it's ready. It'd be nice to have some languages available even before the release👌

**Comment by onein528** [comment]
Really?!

Yeah I think so. well i’ll do my best if you mentioned.👌

**Comment by onein528** [comment]
To tell the truth, I can only translate. I only know C language, and I don't know how to handle uwp. I'm sorry very much.😣

**Comment by MarcAnt01** [comment]
> To tell the truth, I can only translate. I only know C language, and I don't know how to handle uwp. I'm sorry very much.😣

No problem, the app should already be ready to be translated, but I still have to add some strings. Once it's ready, I'll give you a guide to include Japanese

**Comment by onein528** [comment]
thank you.

**Comment by binamracode** [comment]
I can also help with other languages like Hindi, Nepali. if you need

**Comment by onein528** [comment]
It's good news!

**Comment by MarcAnt01** [comment]
Only the owner of the repo can, done anyway

**Comment by onein528** [comment]
Would you like to make the delete button below this photo red?
![Screenshot](https://user-images.githubusercontent.com/62196528/118631863-de147b00-b80a-11eb-8f6d-3c22b6848b5c.png)


I thought it shouldn't be told you in this issue, but it's not a big problem, so I'll told you it here.

**Comment by MarcAnt01** [comment]
I'm not really sure, since currently I like to use default brushes/accent color only, but I'll take that into account

**Comment by MarcAnt01** [comment]
As for the resources, I'm really sorry to be taking so much, but I got irl issues and haven't been working on the app for a long time

**Comment by onein528** [comment]
As Follow:
![Button-mokup](https://user-images.githubusercontent.com/62196528/118633023-fcc74180-b80b-11eb-8bb4-cee75b103f04.png)

> I like to use default brushes/accent color only

Yeah, I think so😊. I only want to use default.

---
That's okay! !! Please instruct at anytime! !!

**Comment by MarcAnt01** [comment]
Hello, I'm really sorry but I had some issues on the localization and I cannot release translations in 2.0 as I hoped to, I'll let you know one I manage to fix this issue :(
This will be done for sure with 2.1 but I have no ETA to give you unfortunately

**Comment by onein528** [comment]
Anytime is fine!

**Comment by villisek** [comment]
Hello,

I would be happy to translate in French when the localization system is ready ;)

>  This will be done for sure with 2.1 but I have no ETA to give you unfortunately

Don't worry ;) IRL is rather a priority than this ^_^

**Comment by MarcAnt01** [comment]
Hello, I'm finally done. Here you go 😄 
https://github.com/MarcAnt01/Fluent-Screen-Recorder/blob/master/translations.md
